src/test/java/org/minions/devfund/marcos/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,110 @@
+package org.minions.devfund.marcos.moviedatabase;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * It is in charged to define the unit tests for Movie Database class.
+ */
+public class MovieDatabaseTest {
+
+    private static final int FIRST_MOVIE_INDEX = 0;
+    private static final double DELTA = 0;
+
+    /**
+     * New Movie Database object has empty actors and movies lists.
+     */
+    @Test
+    public void testNewMovieDatabaseHasEmptyActorsAndMoviesLists() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        assertTrue(movieDatabase.getMovieList().isEmpty());
+        assertTrue(movieDatabase.getActorList().isEmpty());
+    }
+
+    /**
+     * Add new movie object to database.
+     */
+    @Test
+    public void testAddMovieToDatabase() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final String movieTitle = "Interstellar";
+        final String[] actors = new String[]{"Anne Hathaway", "Matthew McConaughey"};
+        final int expectedMovieListSize = 1;
+        final int expectedActorsListSize = 2;
+        movieDatabase.addMovie(movieTitle, actors);
+        assertEquals(expectedMovieListSize, movieDatabase.getMovieList().size());
+        assertEquals(expectedActorsListSize, movieDatabase.getActorList().size());
+    }
+
+    /**
+     * Add duplicated movie object to database.
+     */
+    @Test
+    public void testAddDuplicatedMovieToDatabase() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final String movieTitle = "Interstellar";
+        final String[] actors = new String[]{"Anne Hathaway", "Matthew McConaughey"};
+        final int expectedMovieListSize = 1;
+        final int expectedActorsListSize = 2;
+        movieDatabase.addMovie(movieTitle, actors);
+        movieDatabase.addMovie(movieTitle, actors);
+        assertEquals(expectedMovieListSize, movieDatabase.getMovieList().size());
+        assertEquals(expectedActorsListSize, movieDatabase.getActorList().size());
+    }
+
+    /**
+     * Add new movie rating to existing movie object in database.
+     */
+    @Test
+    public void testAddMovieRatingToDatabase() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final String movieTitle = "Bohemian Rhapsody";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Bohemian Rhapsody" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)